nixos/tests/all-tests.nix

```diff
@@ -21,298 +21,298 @@ let
   nat.firewall-conntrack = handleTest ./nat.nix { withFirewall = true; withConntrackHelpers = true; };
   nat.standalone = handleTest ./nat.nix { withFirewall = false; };
-  ndppd = handleTest ./ndppd.nix {};
-  neo4j = handleTest ./neo4j.nix {};
-  nesting = handleTest ./nesting.nix {};
-  netdata = handleTest ./netdata.nix {};
+  #ndppd = handleTest ./ndppd.nix {};
+  #neo4j = handleTest ./neo4j.nix {};
+  #nesting = handleTest ./nesting.nix {};
+  #netdata = handleTest ./netdata.nix {};
   networking.networkd = handleTest ./networking.nix { networkd = true; };
   networking.scripted = handleTest ./networking.nix { networkd = false; };
   # TODO: put in networking.nix after the test becomes more complete
-  networkingProxy = handleTest ./networking-proxy.nix {};
-  nextcloud = handleTest ./nextcloud {};
-  nexus = handleTest ./nexus.nix {};
+  #networkingProxy = handleTest ./networking-proxy.nix {};
+  #nextcloud = handleTest ./nextcloud {};
+  #nexus = handleTest ./nexus.nix {};
   # TODO: Test nfsv3 + Kerberos
   nfs3 = handleTest ./nfs { version = 3; };
   nfs4 = handleTest ./nfs { version = 4; };
-  nghttpx = handleTest ./nghttpx.nix {};
+  #nghttpx = handleTest ./nghttpx.nix {};
   nginx = handleTest ./nginx.nix {};
-  nginx-etag = handleTest ./nginx-etag.nix {};
-  nginx-sso = handleTest ./nginx-sso.nix {};
+  #nginx-etag = handleTest ./nginx-etag.nix {};
+  #nginx-sso = handleTest ./nginx-sso.nix {};
   nix-ssh-serve = handleTest ./nix-ssh-serve.nix {};
   nixos-generate-config = handleTest ./nixos-generate-config.nix {};
-  novacomd = handleTestOn ["x86_64-linux"] ./novacomd.nix {};
-  nsd = handleTest ./nsd.nix {};
-  nzbget = handleTest ./nzbget.nix {};
-  openarena = handleTest ./openarena.nix {};
+  #novacomd = handleTestOn ["x86_64-linux"] ./novacomd.nix {};
+  #nsd = handleTest ./nsd.nix {};
+  #nzbget = handleTest ./nzbget.nix {};
+  #openarena = handleTest ./openarena.nix {};
   openldap = handleTest ./openldap.nix {};
-  opensmtpd = handleTest ./opensmtpd.nix {};
+  #opensmtpd = handleTest ./opensmtpd.nix {};
   openssh = handleTest ./openssh.nix {};
-  openstack-image-userdata = (handleTestOn ["x86_64-linux"] ./openstack-image.nix {}).userdata or {};
-  openstack-image-metadata = (handleTestOn ["x86_64-linux"] ./openstack-image.nix {}).metadata or {};
-  orangefs = handleTest ./orangefs.nix {};
-  os-prober = handleTestOn ["x86_64-linux"] ./os-prober.nix {};
-  osrm-backend = handleTest ./osrm-backend.nix {};
-  overlayfs = handleTest ./overlayfs.nix {};
-  packagekit = handleTest ./packagekit.nix {};
-  pam-oath-login = handleTest ./pam-oath-login.nix {};
-  pam-u2f = handleTest ./pam-u2f.nix {};
+  #openstack-image-userdata = (handleTestOn ["x86_64-linux"] ./openstack-image.nix {}).userdata or {};
+  #openstack-image-metadata = (handleTestOn ["x86_64-linux"] ./openstack-image.nix {}).metadata or {};
+  #orangefs = handleTest ./orangefs.nix {};
+  #os-prober = handleTestOn ["x86_64-linux"] ./os-prober.nix {};
```

Comment:
it scares me to not run this continuously, but it's actually broken so nvm https://hydra.nixos.org/job/nixos/trunk-combined/nixos.tests.os-prober.x86_64-linux

Reply:
https://github.com/NixOS/nixpkgs/issues/80041